src/Calendar.js

```diff
@@ -11,7 +11,30 @@ import './Calendar.css';
     const WEEKDAYS_SHORT = ['S','M', 'T', 'W', 'T', 'F', 'S'];
 
     // Change format Day with Leading zero
-    // const MyDate = new Date();
+
+    const lunch = {
+        25: ['대리님과 점심'],
+        28: ['멘토님과 상담'],
+    };
+    
+    function renderDay(day) {
+        const date = day.getDate();
+        const cellStyle = {
+        height: 50,
+        width: 100,
+        position: 'relative',
+        };
+        return (
+            <div style={cellStyle}>
+            <div>{date}</div>
+            {lunch[date] &&
```

Comment:
요기 date에 일정이 없을 경우에 undefined key 에러가 뜰거라 옵셔널 적용해주세요.
```suggestion
            {lunch?.[date] &&
```